packages/manager/modules/pci/src/projects/project/storages/databases/components/engines-list/engines-list.html

```diff
@@ -6,12 +6,19 @@
         data-model="$ctrl.selectedEngine"
         data-description="{{:: engine.description }}"
         data-picture="{{:: $ctrl.ENGINE_LOGOS[engine.name] }}"
-        data-label="{{:: $ctrl.capitalize(engine.name) }}"
         data-values="[engine]"
         data-on-change="$ctrl.onChange({ selectedEngine: modelValue })"
         data-disabled="$ctrl.isEngineDisabled(engine)"
         data-required
     >
+        <oui-select-picker-label>
+            <span data-ng-bind=":: $ctrl.capitalize(engine.name)"></span>
+            <span
+                data-ng-if="engine.status === 'BETA'"
```

Comment:
can you move the constants to constant file?

Reply:
Fixed: https://github.com/ovh/manager/pull/5586/commits/f4c7cf9487f1745c9bc94a262c785528601b64f5